test/tests/mfem/kernels/tests

```diff
@@ -206,6 +206,28 @@
     recover = false
     restep = false
   []
+  [MFEMComplex]
+    type = XMLDiff
+    input = complex.i
+    xmldiff = 'OutputData/Complex/Run0/Run0.pvd
+                OutputData/Complex/Run0/Cycle000001/proc000000.vtu'
+    ignored_items = "root['VTKFile']['@version']"
+    requirement = 'The system shall have the ability to solve a complex-valued problem using MFEM.'
+    capabilities = 'mfem'
+    max_parallel = 1
+    recover = false
+  []
+  [MFEMComplexWaveguide]
```

Comment:
The `MFEMComplexWaveguide` test takes a long time to complete the XMLDiff - could this be updated to a check on a figure of merit (relevant postprocessor value) instead?

This would also allow the `max_parallel` constraint to be removed